engine/release-notes/index.md

```diff
@@ -10,932 +10,199 @@ redirect_from:
+  - LCOW was introduced as a technical preview in 17.09 and deprecated in 20.10.
+  - [Deprecation notice](../deprecated.md#linux-containers-on-windows-lcow-experimental)
+- Removed daemon options related to legacy overlay networks used with standalone Swarm.
+  - Removed `--cluster-xx` options from `dockerd`. [moby/moby#40383](https://github.com/moby/moby/issues/40383)
+  - Removed `host-discovery` and overlay networks with external k/v stores. [moby/moby#42247](https://github.com/moby/moby/pull/42247)
+  - [Deprecation notice](../deprecated.md#classic-swarm-and-overlay-networks-using-cluster-store)
+- Removed a deprecated `arm` platform fallback. `--platform linux/arm/vY` will now return a error when `arm/vY` isn't available instead of pulling the wrong image. [moby/moby#44414](https://github.com/moby/moby/pull/44414)
+- Removed the deprecated `SetCustomHTTPHeaders()`, `CustomHTTPHeaders()` options-setters from the Go client API. [moby/moby#42694](https://github.com/moby/moby/pull/42694)
+- Removed the deprecated `WithDialer()` option-setter from the Go client API. [moby/moby#44022](https://github.com/moby/moby/pull/44022)
+  - Users should use `WithDialContext()` instead.
+- Removed the daemon implementation of `opts.QuotedString`. The implementation has moved to the CLI. [moby/moby#43250](https://github.com/moby/moby/pull/43250)
+- Removed separate daemon ID from trust-key in the daemon, and disable generating the trust-key. [moby/moby#43555](https://github.com/moby/moby/pull/43555)
+- API: Removed the deprecated `KernelMemory` option from `POST /containers/create` on API version >= 1.42. [moby/moby#43214](https://github.com/moby/moby/pull/43214)
+  - [Deprecation notice](../deprecated.md#kernel-memory-limit)
+
+### Deprecated
+
+- Require Windows Server RS5 / LTSC 2019 (build 17763) as the minimum to run the daemon. [moby/moby#43254](https://github.com/moby/moby/pull/43254)
+- Deprecated `BuilderSize` in API versions >= 1.42 [moby/moby#42608](https://github.com/moby/moby/pull/42608)
+- Deprecated `BuildCache.Parent` in favor of the newly introduced `BuildCache.Parents` in API version >= 1.42. [moby/moby#43908](https://github.com/moby/moby/pull/43908)
+- Deprecated `pkg/urlutil`, moving the implementation to `builder/remotecontext/urlutil`. [moby/moby#43477](https://github.com/moby/moby/pull/43477)
+
+### Upgrades
+
+- Upgrade Go to `1.19.5`. [docker/cli#3958](https://github.com/docker/cli/pull/3958), [moby/moby#44794](https://github.com/moby/moby/pull/44794)
+- Upgrade `rootlesskit` to `v0.14.4`. [moby/moby#42708](https://github.com/moby/moby/pull/42708)
+- Upgrade `buildkit` to `v0.10.0`. [moby/moby#43239](https://github.com/moby/moby/pull/43239)
```

Comment:
It's BuildKit 0.10.6 actually: https://github.com/moby/moby/blob/de3f216113f27708006923376826686db3783ed4/vendor.mod#LL52C27-L52C34

```suggestion
- Upgrade `buildkit` to `v0.10.6`. [moby/moby#43239](https://github.com/moby/moby/pull/43239)
```

Not sure if we should put all issues for each update. I guess the first one introducing BuildKit 0.10 is fine.